agent/bench-scripts/pbench-uperf

```diff
@@ -549,6 +557,10 @@ for protocol in `echo $protocols | sed -e s/,/" "/g`; do
 									if [ -z "$client" ]; then
 										client="127.0.0.1"
 									fi
+                                    uperf_server=`echo $uperf_servers | cut -d, -f$server_nr`
+                                    if [ -e "$uperf_server" ]; then
+                                        uperf_server="$server"
+                                    fi
```

Comment:
The surrounding code uses hard-tabs for indentation; the new code is using spaces.  Please match the surrounding code.  (Ditto for the other additions.)

Reply:
oups, incorrectly configured vim. Sorry about that.